### DISCUSSION TITLE
There used to be a file called ContractTileFormatting.json

### DISCUSSION THREAD

**Discussion by HenrikMozartDXC** [pnp/syntex-samples#23] [Q&A] [answered]
Question:

There used to be a file called ContractTileFormatting.json, where can I find that file now?

It was a great template to for visualisation in Teams, and is also referenced in: https://learn.microsoft.com/en-us/microsoft-365/contentunderstanding/solution-manage-contracts-step2#attach-your-sharepoint-document-library-to-the-contracts-tab

I would love to get a copy of that json or similar but cannot find it anywhere.

Best regards Henrik

**Comment by LeonArmston** [comment, marked as answer]
Hi @HenrikMozartDXC 

Due to the repository being re-organised to make it web friendly - the url has changed to [https://github.com/pnp/syntex-samples/blob/main/scenario-samples/Contracts-Management/View%20Formatter/ContractTileFormatting.json](https://github.com/pnp/syntex-samples/blob/main/scenario-samples/Contracts-Management/View%20Formatter/ContractTileFormatting.json)

The MS Docs page has also been updated.
